pkg/passphrase/passphrase.go

```diff
@@ -46,23 +46,35 @@ var (
 	// ErrTooManyAttempts is returned if the maximum number of passphrase
 	// entry attempts is reached.
 	ErrTooManyAttempts = errors.New("Too many attempts")
+
+	defaultAliasMap = map[string]string{
+		"root":       "root",
+		"targets":    "targets",
+		"snapshot":   "snapshot",
+		"timestamps": "timestamps",
+	}
```

Comment:
This seems kind of silly. Why not just avoid the alias map lookup if aliasMap is nil?